tests/integration/llm/prepare.py

```diff
@@ -896,6 +896,12 @@
         "option.max_rolling_batch_size": 32,
         "option.output_formatter": "jsonlines"
     },
+    "flan-t5-xxl": {
+        "engine": "MPI",
+        "option.model_id": "s3://djl-llm/flan-t5-xxl-trtllm-compiled/",
```

Comment:
you need to version it, please add 0.8.0 as part of the key. Just like the pre-compiled falcon model

Reply:
Done